src/BugReporting.jl

```diff
@@ -112,10 +123,7 @@ include("sync_compat.jl")
 
 function upload_rr_trace(trace_directory)
     # Auto-pack this trace directory if it hasn't already been:
-    if !is_packed(trace_directory)
-        @info("Automatically calling `rr_pack()` on $(trace_directory) before upload...")
-        rr_pack(trace_directory)
-    end
+    rr_pack(trace_directory)
```

Comment:
This doesn't work since the artifact directory is made read-only after creation.